unified-runtime/test/conformance/device_code/mean.cpp

```diff
@@ -27,6 +27,7 @@ int main() {
         num_groups, group_size, [=](sycl::group<1> group) {
           auto group_id = group.get_group_id();
           group.parallel_for_work_item([&](sycl::h_item<1> item) {
+            auto group_id = group.get_group_id();
```

Comment:
looks weird so we should have an explanatory comment

Reply:
I've added a comment although I'm not 100% sure what the issue is.